.circleci/config.yml

```diff
@@ -5,6 +5,8 @@ jobs:
   build_and_test:
     docker:
       - image: circleci/node:8.11.2@sha256:495a13db597b5306f05ae728489fedb9faa4defbac24ca0e00e8cc1eca95396d
+    environment:
+      ADDON_URL: https://addons.mozilla.org/firefox/downloads/latest/firefox-color/
```

Comment:
Although it doesn't really matter for the validity of the URL, let's add "firefox-color.xpi" so that the link ends with a ".xpi" extension.